index.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>"HTML,CSS&GitBasics"</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Muli:400,600,600i" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    <h1>Junior developer’s resume</h1>
+    <div class="blueLine"></div>
```

Comment:
I would suggest you using the more appropriate class name here (for example "headerSeparator"). Within the class name, you shouldn't describe appearance. If you change the color of this line, you will need to change the class name as well. 
```suggestion
    <div class="headerSeparator"></div>
```